modules/nginxvts/testdata/vts.json

```diff
@@ -0,0 +1,44 @@
+{
+  "hostName":"Web",

```

Comment:
is it full response? or you removed some data we are not using currently?

Reply:
this is not the full response. Response will be different according to nginx vts configuration. This  should be enough for the "total" version of nginxvts plugin currently.

Reply:
> Response will be different according to nginx vts configuration.

could you share used configuration?

Reply:
The configuration is very simple, just modify nginx.conf:
```conf
http {
    vhost_traffic_status_zone;
    ...
    server {
        ...

        location /status {
            vhost_traffic_status_display;
            vhost_traffic_status_display_format html;
        }
    }
}
```
The _vhost_traffic_status_display_format_ can be set to _json_,，I use _html_ for debug purpose.